style.css

```diff
@@ -0,0 +1,1547 @@
+    margin-top: 26px;
+    font-family: "Inter", serif;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 20px;
+    text-align: center;
+    color: white;
+    display: flex;
+  }
+
+  .buttons-up {
+    float: right;
+    margin-right: 156px;
+    margin-top: -20px;
+  }
+
+  .buttons-up ul {
+    list-style: none;
+    display: flex;
+    text-decoration: none;
+    color: white;
+    gap: 32px;
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .buttons-up ul li a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .union {
+    display: none;
+  }
+
+  /* .headline {
+    width: 100%;
+    height: 870px;
+    background-color: #1c1a19;
+    background-image: url("icons/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position-x: right;
+
+  } */
+
+  .headline {
+    display: none;
+  }
+
+  .alex {
+    width: 100%;
+    height: 160px;
+    padding-left: 143px;
+    padding-top: 264px;
+    font-family: "Crete Round", serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    text-align: left;
+    color: #f55800;
+  }
+
+  .image-head {
+    float: right;
+    margin-top: -450px;
+    width: 100%;
+    max-width: fit-content;
+  }
+
+  .soft {
+    font-family: "Inter", serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    text-align: left;
+    color: #fff;
+    padding-top: 170px;
+    padding-left: 143px;
+    width: 100%;
+    height: 24px;
+  }
+
+  .text {
+    font-family: "Inter", serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    text-align: left;
+    color: #f4f5f7;
+    width: 700px;
+    height: 566px;
+    padding-left: 143px;
+    padding-top: 60px;
+    max-width: fit-content;
+  }
+
+  .grid-headline {
+    display: grid;
+    grid-template-columns: (2, 1 fr);
+    background-color: #1c1a19;
+    height: 870px;
+  }
+
+  .icon-desk {
+    list-style: none;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    margin-top: -650px;
+    padding-left: 18px;
+    gap: 15px;
+  }
+
+  #work {
+    width: 100%;
+    display: grid;
+    max-width: fit-content;
+  }
+
+  .recent-works {
+    font-family: "Crete Round", serif;
+    font-size: 40px;
+    font-weight: 400;
+    line-height: 52px;
+    text-align: left;
+    color: #091e42;
+    padding-left: 153px;
+    padding-top: 135px;
+    width: 100%;
+  }
+
+  .vector {
+    display: none;
+  }
+
+  .gridContainer {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 20px;
+    height: 448px;
+    margin: 60px 50px 0 70px;
+  }
+
+  .container2 {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 8px;
+
+    /* margin: 20px 60px 60px 70px; */
+    height: 907px;
+    width: 100%;
+
+    /* max-width: fit-content; */
+
+    /* margin-left: 6%; */
+
+    /* margin-right: 20px; */
+
+    /* margin-top: 20px; */
```

Comment:
- [ ] For clean code and best practices reasons, kindly make sure you remove all commented and unused code lines from your code base